blocks/card-list-block/features/card-list/default.jsx

```diff
@@ -108,95 +132,130 @@ const CardListItems = (props) => {
 		contentItems = contentItems.slice(0, displayAmount);
 	}
 
-	const Wrapper = title ? HeadingSection : React.Fragment;
+	const sourceContent = contentElements[0];
+	const phrases = getTranslatedPhrases(getProperties(arcSite).locale || "en");
 
-	const showSeparator = !!(
-		contentItems[0] &&
-		contentItems[0].credits &&
-		contentItems[0].credits.by &&
-		contentItems[0].credits.by.length !== 0
-	);
+	const displayDate = localizeDate(sourceContent.display_date, dateFormat, language, timeZone);
+
+	/* Author Formatting */
+	const bylineNodes = formatAuthors(sourceContent?.credits?.by, phrases.t("byline-block.and-text"));
+	const hasAuthor = !!bylineNodes.length;
+
+	// Start Overline data
+	const {
+		display: labelDisplay,
+		url: labelUrl,
+		text: labelText,
+	} = (sourceContent.label && sourceContent.label.basic) || {};
+	const shouldUseLabel = !!labelDisplay;
+
+	const { _id: sectionUrl, name: sectionText } =
+		(sourceContent.websites &&
+			sourceContent.websites[arcSite] &&
+			sourceContent.websites[arcSite].website_section) ||
+		{};
+
+	// Default to websites object data
+	let [text, url] = [sectionText, sectionUrl];
+
+	if (sourceContent?.owner?.sponsored) {
+		text = sourceContent?.label?.basic?.text || phrases.t("overline.sponsored-content");
+		url = null;
+	} else if (shouldUseLabel) {
+		[text, url] = [labelText, labelUrl];
+	}
+	// End Overline data
 
 	return contentItems.length > 0 ? (
 		<HeadingSection>
-			<div className="card-list-container">
-				{title ? <Heading className="card-list-title">{title}</Heading> : null}
-				<Wrapper>
-					<article
-						className="list-item-simple"
-						key={`card-list-${contentItems[0].websites[arcSite].website_url}`}
+			<Stack className={BLOCK_CLASS_NAME}>
+				{title ? <Heading className={`${BLOCK_CLASS_NAME}__title`}>{title}</Heading> : null}
```

Comment:
Looks like the conditional `Wrapper` logic has been removed - if there is a title we need another `HeadingSection` to increment the heading levels below

Reply:
oh damn -- will add back

Reply:
updated